IronFrame.Test/IronFrame.Test.csproj

```diff
@@ -33,6 +33,10 @@
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="Interop.NetFwTypeLib">
```

Comment:
I tried to remove this line and now neither`build build` nor`devenv IronFrame.sln /build "Release"` is working.